cluster/gce/config-test.sh

```diff
@@ -161,7 +161,7 @@ KUBE_UP_AUTOMATIC_CLEANUP=${KUBE_UP_AUTOMATIC_CLEANUP:-false}
 TEST_CLUSTER="${TEST_CLUSTER:-true}"
 
 # Storage backend. 'etcd2' supported, 'etcd3' experimental.
-STORAGE_BACKEND=${STORAGE_BACKEND:-etcd2}
+STORAGE_BACKEND=${STORAGE_BACKEND:-etcd3}
```

Comment:
You mean with just this change (and without the changes to etcd.manifest?
I set-up Jenkins project for such configuration today and it works.